src/test/java/org/folio/logintest/LoginWithExpiryTest.java

```diff
@@ -0,0 +1,300 @@
+        .spec(spec)
+        .body(credsObject3.encode())
+        .when()
+        .post(CRED_PATH)
+        .then()
+        .log().all()
+        .statusCode(201);
+
+    // The credentials object doesn't have a id so it is not considered active.
+    RestAssured.given()
+        .spec(spec)
+        .body(credsObject3.encode())
+        .when()
+        .post(LOGIN_WITH_EXPIRY_PATH)
+        .then()
+        .log().all()
+        .statusCode(422)
+        .contentType("application/json")
+        .body("errors[0].code", is("user.blocked"))
+        .body("errors[0].message", is("User must be flagged as active"));
+
+    // The following credentials objects have incorrect passwords.
+    // One provides the userId and the other provides the username, but neither
+    // should work.
+    RestAssured.given()
+        .spec(spec)
+        .body(credsObject4.encode())
+        .when()
+        .post(LOGIN_WITH_EXPIRY_PATH)
+        .then()
+        .log().all()
+        .statusCode(422)
+        .contentType("application/json")
+        .body("errors[0].code", is("password.incorrect"))
+        .body("errors[0].message", is("Password does not match"));
+
+    RestAssured.given()
+        .spec(spec)
+        .body(credsObject5.encode())
+        .when()
+        .post(LOGIN_WITH_EXPIRY_PATH)
+        .then()
+        .log().all()
+        .statusCode(422)
+        .contentType("application/json")
+        .body("errors[0].code", is("password.incorrect.block.user"))
+        .body("errors[0].message", is("Fifth failed attempt"));
+
+    // Now we update our credentials object with a new password and try again.
+    RestAssured.given()
+        .spec(spec)
+        .body(credsObject6.encode())
+        .when()
+        .post(UPDATE_PATH)
+        .then()
+        .log().all()
+        .statusCode(204);
+
+    // These should now succeed.
+    testCookieResponse(credsObject4);
+    testCookieResponse(credsObject5);
+  }
+
+  private void testCookieResponse(JsonObject creds) {
+    RestAssured.given()
+        .spec(spec)
+        .body(creds.encode())
+        .when()
+        .post(LOGIN_WITH_EXPIRY_PATH)
+        .then()
+        .log().all()
+        .statusCode(201)
+        .contentType("application/json")
+        .cookie(LoginAPI.REFRESH_TOKEN, RestAssuredMatchers.detailedCookie()
+            .value(Mocks.REFRESH_TOKEN)
+            // Account for time drift because we're using Now.Instant to compute max-age.
+            .maxAge(allOf(greaterThan(Mocks.REFRESH_TOKEN_EXPIRATION - 2), lessThan(Mocks.REFRESH_TOKEN_EXPIRATION + 1)))
+            .path("/authn")
+            .httpOnly(true)
+            .secured(true))
+        .cookie(LoginAPI.ACCESS_TOKEN, RestAssuredMatchers.detailedCookie()
+            .value(Mocks.ACCESS_TOKEN)
+            // Account for time drift because we're using Now.Instant to compute max-age.
+            .maxAge(allOf(greaterThan(Mocks.ACCESS_TOKEN_EXPIRATION - 2), lessThan(Mocks.REFRESH_TOKEN_EXPIRATION + 1)))
```

Comment:
Please add

    .domain((String) null)  // not setting Domain disables subdomains
    .path((String) null)    // not setting Path allows all paths